webpack/scenes/ContentViews/Details/ComponentContentViews/ComponentVersion.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { urlBuilder } from 'foremanReact/common/urlHelpers';
+import { translate as __ } from 'foremanReact/common/I18n';
+import {
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+
+const ComponentVersion = ({ componentCV }) => {
+  const { latest, content_view_version: cvVersion, content_view: cv } = componentCV;
+  const {
+    id,
+    latest_version: latestVersion,
+  } = cv;
+  const { version } = cvVersion;
+  const latestDescription = __('Latest(automatically updates)');
+  const manualVersionText = (latestVersion === version) ? __('Latest Version') : __(`New Version is available: Version ${latestVersion}`);
```

Comment:
Is this the wording we decided on?  If so, only the first words should be capitalized:
```suggestion
  const manualVersionText = (latestVersion === version) ? __('Latest version') : __(`New version is available: Version ${latestVersion}`);
```

Reply:
Updated.. 👍🏽